crates/build_utils/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "build_utils"
```

Comment:
Small comment: [`cargo-deny` is failing](https://ci.iog.io/build/8575095/nixlog/1), because these new `Cargo.toml`s lack license information:

```
error[unlicensed]: build_utils = 0.1.0 is unlicensed
  ┌─ path+file:///build/source/crates/build_utils#0.1.0:2:9
  │
2 │ name = "build_utils"
  │         ━━━━━━━━━━━ a valid license expression could not be retrieved for the crate
3 │ version = "0.1.0"
4 │ license = ""
  │            ─ license expression was not specified
  │
  ├ build_utils v0.1.0
    ├── (build) blockfrost-platform v0.0.2
    └── (build) common v0.0.1
        ├── blockfrost-platform v0.0.2 (*)
        ├── dolos v0.0.1
        │   └── blockfrost-platform v0.0.2 (*)
        └── node v0.1.0
            └── blockfrost-platform v0.0.2 (*)
```